### PR TITLE
Whitespace

### PR DESCRIPTION
**Pull request by playerskel**
Not much to say really.

### CODE REVIEW

File: include/z64camera.h [hunk 1]
```diff
@@ -391,7 +391,7 @@ typedef struct {
     { groundAtLerpStepScale, CAM_DATA_GROUND_AT_LERP_STEP_SCALE }
 
 typedef struct {
-    
+
```
Comment (Dragorn421) [inline include/z64camera.h:394]:
```suggestion
```

File: src/overlays/actors/ovl_En_Fz/z_en_fz.h [hunk 2]
```diff
@@ -13,14 +13,14 @@ typedef struct {
...
-    /* 0x0010 */ Vec3f velocity; 
+    /* 0x0010 */ Vec3f velocity;
     /* 0x001C */ Vec3f accel;
     /* 0x0028 */ char unk_28[0x4];
     /* 0x002C */ s16 primAlpha; // transparency in RGBA colour system
     /* 0x002E */ s16 primAlphaState; // 0: increasing (more opaque) 1: decreasing (more transparent) 2: collision
-    /* 0x0030 */ f32 xyScale; // 
-    /* 0x0034 */ f32 xyScaleTarget; 
-    /* 0x0038 */ u8 isTimerMod8; // conditional, used to run CollisionCheck_SetAT 
+    /* 0x0030 */ f32 xyScale; //
```
Comment (Dragorn421) [inline src/overlays/actors/ovl_En_Fz/z_en_fz.h:21]:
```suggestion
    /* 0x0030 */ f32 xyScale;
```

**Review by Dragorn421 — CHANGES_REQUESTED** [review]

**Review by fig02 — APPROVED** [review]
after the changes above are addressed this is fine for me.

some changes in here were not whitespace related, though im sure it was an accident when splitting up your bigger pr. just something to keep in mind in general though. its fine that they stay in

File: src/overlays/actors/ovl_En_Fz/z_en_fz.h (hunk 2 above)
Comment (playerskel) [reply src/overlays/actors/ovl_En_Fz/z_en_fz.h]: Done.

File: include/z64camera.h (hunk 1 above)
Comment (playerskel) [reply include/z64camera.h]: Done.

**Review by Dragorn421 — APPROVED** [review]

**Review by Roman971 — APPROVED** [review]

**Pull request merged**